dev-python/icalendar/icalendar-4.0.6.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="Package used for parsing and generating iCalendar files (RFC 2445)"
+HOMEPAGE="https://github.com/collective/icalendar"
+SRC_URI="mirror://pypi/i/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 ~arm ~arm64 x86"
+DOCS=( "README.rst" )
```

Comment:
Do you really need it? If so, please put it below *DEPEND and other standard vars, otherwise it's easy to miss.

Reply:
No idea. It was there in 4.0.3; I didn’t even read the README, rather just assumed the original maintainer had a reason for adding it. It doesn’t look very useful (it looks like it’s aimed at prospective users, not current users). Deleted in d98f38aa7cce.